spec/queries/search_results_query_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SearchResultsQuery, type: :model do
```

Comment:
Not sure if the type is `model`? 🤔

Reply:
This should be correct: https://github.com/nimblehq/travelbook-web/blob/development/spec/queries/travelogues_query_spec.rb 💪

Reply:
@Tuubz Yeah... I am confused about this, seems we need the `:type` when having some config before each cases, on ewapayroll, they don't have type for query spec files 😄 

https://github.com/nimblehq/ewa-payroll-web/blob/049a5896bc4e172cfee496e8e61f58af4b1e4e38/spec/queries/holiday_query_spec.rb#L4